src/mocks/handlers/psApiHandlers.ts

```diff
@@ -54,7 +58,10 @@ export const psApiHandlers = [
       return submissionsResponse(mockSubmissions.aimsAll);
     }
 
-    return submissionsResponse(mockSubmissions.daartHl7);
+    if (dataStreamId == "daart")
```

Comment:
Why do you need to handle this special case?

Reply:
This is the way the mocks are setup. I added the `ehdi` and `eicr` datastreams to the mocked user datastreams. Without this blank return for things besides `daart` and `aims-celr` the dashboard wont render a realistic set of counts. Other option would be to add mocked data for `eicr` and `ehdi`, but that is more time for mocking.